README.md

```diff
@@ -438,10 +438,10 @@ desc "Return super-secret information", {
 <a name="hiding" />
 #### Hiding an Endpoint
 
-You can hide an endpoint by adding ```hidden: true``` in the description of the endpoint:
+You can hide an endpoint by adding ```hidden: true``` on the verb method of the endpoint, such as `get`, `post` and `put`:
```

Comment:
@thogg4 … thanks for contribution …
please don't replace existed documentation, it is still correct, only add as additional possibility to hide an endpoint